src/sql/workbench/parts/notebook/browser/cellToggleMoreActions.ts

```diff
@@ -30,10 +30,6 @@ export class CellToggleMoreActions {
 		@IInstantiationService private instantiationService: IInstantiationService) {
 		this._actions.push(
 			instantiationService.createInstance(DeleteCellAction, 'delete', localize('delete', "Delete")),
-			instantiationService.createInstance(AddCellFromContextAction, 'codeBefore', localize('codeBefore', "Insert Code Before"), CellTypes.Code, false),
```

Comment:
Can we create a task to track need for keyboard shortcuts to do these actions? Moving from right-click to new inline set of actions, makes it harder for accessibility. We can either update Add Cell command or add new accelerators for all these.

Again: not needed in this PR, but we should track & prioritize doing this.

Reply:
Doesn't look like we had these in the right click menu, just the overflow menu in the corner of the cell. We have existing shortcuts for adding new cells: Ctrl+Shift+C for code, Ctrl+Shift+T for text.